### DISCUSSION TITLE
Configure Docker images to ignore GitHub Pre-releases

### DISCUSSION THREAD

**Discussion by calvinbui** [renovatebot/renovate#27923] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitea

### Please tell us more about your question or problem

Hey 👋 

Renovate is making PRs for updates to my Mattermost running in Docker.

![image](https://github.com/renovatebot/renovate/assets/3604363/f48b1430-ee69-42b7-b194-805a49ec722e)

The new version, [v9.6.0](https://github.com/mattermost/mattermost/releases/tag/v9.6.0) is considered a `pre-release` on GitHub, but as I understand, [Docker Hub](https://hub.docker.com/layers/mattermost/mattermost-team-edition/9.6.0/images/sha256-afff189b9be52bf3b255ba7de7ca89c0b5cdb9b39164fc2163f82765adcedb46?context=explore) does not set it as a `pre-release.`

Is there anyway to have Renovate see the `pre-release` tag from GitHub and ignore the update until the `pre-release` tag is gone?

![image](https://github.com/renovatebot/renovate/assets/3604363/bbde0c8e-80c4-4ec0-b19b-1ad18ffa3273)

My config:

```json5
    /* mattermost */
    {
      matchPackagePatterns: [".*\\/mattermost-(team|enterprise)-edition"],
      customChangelogUrl: "https://github.com/mattermost/mattermost-server",
      prBodyNotes: "[Mattermost changelog](https://docs.mattermost.com/deploy/mattermost-changelog.html)",
    },
```

**Comment by rarkins** [comment, marked as answer]
No. They really shouldn't publish stable tags as prereleases like that
